impeller/renderer/backend/vulkan/allocator_vk.cc

```diff
@@ -132,12 +132,27 @@ static constexpr vk::ImageUsageFlags ToVKImageUsageFlags(PixelFormat format,
 
   if (usage & static_cast<TextureUsageMask>(TextureUsage::kShaderRead)) {
     vk_usage |= vk::ImageUsageFlagBits::eSampled;
+    // unmark the usage as transient if it is used as a sampled image
```

Comment:
We can leave out the issue link as that will get outdate. Instead a comment here saying: "Device transient images can only be used as attachments. The caller specified incorrect usage flags and is attempting to read a device transient image in a shader. Unset the transient attachment flag.".

Maybe even add a validation log?

Reply:
i'm leaving the reference to the bug and added a comment there to go back and fix the default to be `kUnknown`